src/pages/graphql/schema/cart/mutations/place-order.md

```diff
@@ -4,7 +4,7 @@ title: placeOrder mutation
 
 # placeOrder mutation
 
-The `placeOrder` mutation converts the cart into an order and returns an order ID. You cannot manage orders with GraphQL, because orders are part of the backend. You can use REST or SOAP calls to manage orders to their completion.
+The `placeOrder` mutation converts the cart into an order and returns an order ID. You cannot manage orders with GraphQL, because orders are part of the back end. You can use REST or SOAP calls to manage orders to their completion.
```

Comment:
```suggestion
The `placeOrder` mutation converts the cart into an order and returns an order ID. You cannot manage orders with GraphQL, because orders are part of the backend. You can use REST or SOAP calls to manage orders to their completion.
```